### PR TITLE
[needs-docs] Move "map theme" combobox near list of layers in Dxf export dialog

### PR DESCRIPTION
**Pull request by DelazJ**
Switch placement of crs and map theme options because the map theme option directly impacts checked layers in the list, so better have them closer.
Also fix spacing in scale widget for more consistency.
image

### CODE REVIEW

**Comment by nirvn** [comment]
@DelazJ , good move on the scale widget, that was on my TODO list.

**Comment by nirvn** [comment]
@DelazJ , this could benefit from using QgsFileWidget too.

File: src/app/qgsdxfexportdialog.cpp [hunk 1]
```diff
@@ -567,32 +579,22 @@ QgsDxfExport::SymbologyExport QgsDxfExportDialog::symbologyMode() const
...
-  QString filePath = mFileLineEdit->text();
+  QString filePath = mFileName->filePath();
   if ( filePath.isEmpty() )
   {
     btn->setEnabled( false );
+    return;
   }
 
   QFileInfo fi( filePath );
-  btn->setEnabled( fi.absoluteDir().exists() );
```
Comment (DelazJ) [inline src/app/qgsdxfexportdialog.cpp]: Genuine question: why do we check the folder existence? If it doesn't exist yet, shouldn't we create it?

**Comment by nirvn** [comment]
The code looks OK, but I've never used this dialog, I'd like a second opinion on the combo box move. @nyalldawson ?

**Comment by nyalldawson** [comment]
+1 on the rearrangement, much more logical

**Pull request merged**